### PR TITLE
Add summary from June 30th meeting

### PR DESCRIPTION
**Pull request by wengj9**
A bit short, but we breezed through the more "meaty" (benchmark-focused) parts of the paper 😅

### CODE REVIEW

**Comment by wengj9** [comment]
Also just noticed the header links in the Netlify preview link to the live https://systemd.works site and not the preview, so here's a link to my post on the preview: https://deploy-preview-5--systemd-works.netlify.app/posts/vmware-esx-memory-management/

File: content/posts/vmware-esx-memory-management.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### Resource sharing
+
+ESX uses a ticket/lease mechanism to distribute resources to guests - the more tickets a guest has, the more resources available to them. To prevent a guest from hogging/acquiring all resources and refusing to release them, ESX implements the concept of the "idle memory tax"; the tax comes at reclamation time - guests with more unused resources will be picked first to have resources taken away from them.
+
+## Benchmarks
+
+Many benchmarks were done - refer to the paper for methodology, equations, and results. The readers/author of this summary do not have the pre-requisite background to accurately interpret those results.
```
Comment (Ishaandham19) [inline content/posts/vmware-esx-memory-management.md:42]: OS MS student saying this... Get PhD asap 😄

Comment (wengj9) [reply content/posts/vmware-esx-memory-management.md:42]: Yeah I was being lazy 😅 - I'll read the benchmark section of the paper and revise this PR

File: content/posts/vmware-esx-memory-management.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Future Work
+
+In the time since the paper was released, there does not appear to have been much advancement in this area - Linux/KVM [still uses](https://pmhahn.github.io/virtio-balloon/) memory ballooning. At a higher-level, containerization has taken over many of the old use cases of virtual machines and in that case, a regular operating system kernel can rely on traditional OS techniques to do resource management. Additionally, we hypothesize many of these techniques are "good enough" and no one has felt motivated to significantly improve on these techniques.
```
Comment (Ishaandham19) [inline content/posts/vmware-esx-memory-management.md]: Aren't containers mostly run on VMs offered by cloud providers (like GCE)?

File: content/posts/vmware-esx-memory-management.md (hunk 1 above)
Comment (Ishaandham19) [reply content/posts/vmware-esx-memory-management.md:42]: Don't stress it. imo I think this captures the concepts of the paper well.

**Review by Ishaandham19 — APPROVED** [review]
Thanks!

File: content/posts/vmware-esx-memory-management.md (hunk 2 above)
Comment (wengj9) [reply content/posts/vmware-esx-memory-management.md]: Good call-out, many of them are. It seems like the hyperscalers use smaller hypervisors in cases like GCE/GKE - see [gVisor](https://gvisor.dev/docs/) (Google) and [Firecracker](https://firecracker-microvm.github.io/) (Amazon). It looks like they minimize KVM usage/remove it altogether.

**Pull request merged**